src/magic_folder/web.py

```diff
@@ -114,11 +134,158 @@ class APIv1(Resource, object):
     """
     _global_config = attr.ib()
     _global_service = attr.ib()
+    _tahoe_client = attr.ib()
 
     def __attrs_post_init__(self):
         Resource.__init__(self)
         self.putChild(b"magic-folder", MagicFolderAPIv1(self._global_config))
         self.putChild(b"snapshot", SnapshotAPIv1(self._global_config, self._global_service))
+        self.putChild(b"participants", ParticipantsAPIv1(self._global_config, self._tahoe_client))
```

Comment:
nit: Given that we are adding a participant to a folder, I'd be inclined to put this endpoint as a child of the(a) folder endpoint. So something like `/v1/magic-folder/<folder-name>/participants`.

Reply:
I figured I was following the convention of the other things (e.g. `snapshot` does it this way around too). I kind of agree that putting the folder-name higher-up might make more sense -- but we should maybe switch `/snapshot` around too?

I added a "question" ticket about this: https://github.com/LeastAuthority/magic-folder/issues/359 so perhaps could address in a followup (or not, depending on the conclusion ;)